src/services/utilities.ts

```diff
@@ -857,6 +857,13 @@ namespace ts {
         return false;
     }
 
+    export function isInBreakOrContinue(sourceFile: SourceFile, position: number): boolean {

```

Comment:
I would just move this to `completions.ts` since it's unlikely to be needed outside of a completion context.